src/main/java/com/mraof/minestuck/entity/underling/OgreEntity.java

```diff
@@ -93,4 +99,15 @@ public void onDeath(DamageSource cause)
 			}
 		}
 	}
+
+	@Override
+	protected boolean isAppropriateTarget(LivingEntity entity)
+	{
+		if(entity instanceof ServerPlayerEntity)
+		{
+			//Max rung was chosen based off of approximately what rung the player would be at when they can start one or two-hitting this mob. Minimum rung was chosen in order to prevent low-leveled players from being ganged up on while until they are strong enough to take on multiple at a time.
+			return PlayerSavedData.getData((ServerPlayerEntity) entity).getEcheladder().getRung() < 21 && PlayerSavedData.getData((ServerPlayerEntity) entity).getEcheladder().getRung() > 14;
```

Comment:
I don't really like the design choice of having both a lower and an upper range in which they aren't actively targeting the player. I'd rather see a different solution for handling situations with underleveled players.